dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/command/DownloadCommand.java

```diff
@@ -68,8 +68,8 @@
   private boolean force = false;
   @Parameter(names = "--manifest", description = "Path to manifest id, url or file")
   private ManifestResource manifestResource;
-  @Parameter(names = "--object-id", description = "Object id to download", validateValueWith = ObjectIdValidator.class)
-  private String objectId;
+  @Parameter(names = "--object-id", description = "Object id to download", validateValueWith = ObjectIdListValidator.class, variableArity = true)
+  private List<String> objectId = new ArrayList<>();
```

Comment:
I explicitly kept the parameter the same so existing workflows wouldn't have to change. I didn't think it was worth the clutter of having separate --objectId and --objectIds parameters when it's exactly the same code